### DISCUSSION TITLE
Does regular expression search actually work?

### DISCUSSION THREAD

**Discussion by gw8484** [dail8859/NotepadNext#334] [Q&A]
I am trying to use regular expression search on v0.5.6 and it doesn't appear to work at all.

**Comment by dail8859** [comment]
The short answer is, yes. The long answer is advanced/complex regex support is not near as good as Notepad++. It's definitely one area of improvement that is needed. I stuck with the default regular expression implementation that was provided by Qt for now.

If you could give specific examples or screenshots that may help to see what the issue may be.

**Reply by gw8484** [reply]
I tried it again and it seems to work only in smaller documents.  When I do it on a 3MB document it fails with 0 found.
